unit_test/TestTHREADS_Category.hpp

```diff
@@ -0,0 +1,19 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef CABANA_TEST_PTHREAD_CATEGORY_HPP
+#define CABANA_TEST_PTHREAD_CATEGORY_HPP
+
+#define TEST_CATEGORY pthread
```

Comment:
You don't have a `Threads` device type listed in your `CMakeLists.txt` for the unit tests so you probably don't need this file. We haven't been testing against pthreads as everything seems to have moved to OpenMP.

Reply:
Removed